lib/p5.play.js

```diff
@@ -4125,7 +4125,13 @@ p5.prototype._warn = function(message) {
    * @param {p5.Transform2D|Sprite} parent
    */
   p5.CollisionShape.prototype.setParentTransform = function(parent) {
-    if (parent instanceof Sprite) {
+    if (parent instanceof Sprite && this instanceof p5.CircleCollider) {
```

Comment:
Or possibly CircleCollider should override `setParentTransform` - which might be cleaner in this case, since CollisionShape doesn't currently have particular knowledge of CircleCollider, I don't think.